app/src/main/java/com/example/hungrywolfs/model/DetailsViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.hungrywolfs.model
+
+import android.util.Log
+import androidx.lifecycle.*
+import com.example.hungrywolfs.SingleLiveEvent
+import com.example.hungrywolfs.network.MealApi
+import com.example.hungrywolfs.network.MealDetailsInfo
+import kotlinx.coroutines.launch
+
+class DetailsViewModel : ViewModel() {
+
+    private val _navigationToHome = SingleLiveEvent<Any>()
+    val navigationToHome = _navigationToHome
+
+    private var _mealDetails = MutableLiveData<MealDetailsInfo>()
+    val mealDetails: LiveData<MealDetailsInfo> = _mealDetails
+
+    private val _mealTags = MutableLiveData<List<String>>()
+    val mealTags: LiveData<List<String>> = Transformations.map(_mealTags) { it }
```

Comment:
This is not really how this should be handled, you technically map a value to itself without any modifications. This is where you should do the splitTagMeals instead of setting _mealTags.value by hand

Reply:
I tried to solve it but it doesn't work.